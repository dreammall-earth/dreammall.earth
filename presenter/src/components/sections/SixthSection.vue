<template>
  <div class="section6 v-container pa-16">
    <v-row class="section-header">
      <v-col>
        <h2 class="section-headline">
          <span class="prepend">{{ $t('home.section6.headline') }}</span
          >&nbsp;
          <span class="append">{{ $t('home.section6.headlineAppend') }}</span>
        </h2>
      </v-col>
    </v-row>
    <v-row class="section-content">
      <v-col>
        <p>
          {{ $t('home.section6.content') }}
        </p>
      </v-col>
    </v-row>
    <v-row class="section-footer text-center">
      <v-col align-self="center">
        <MainButton
          class="mt-8"
          :label="$t('home.section6.buttonText')"
          size="large"
          variant="primary"
          href="/#contactname"
          >{{ $t('home.section6.buttonText') }}</MainButton
        >
      </v-col>
    </v-row>
  </div>
</template>

<script lang="ts" setup>
import MainButton from '#components/inputs/MainButton.vue'
</script>

<style scoped lang="scss">
.section6 {
  max-width: 960px;

  .section-header {
    text-align: center;

    h2.section-headline {
      font-family: Poppins, sans-serif;
      font-size: 2.25rem;
      font-style: normal;
      font-weight: 700;
      line-height: normal;
      letter-spacing: -0.045rem;

      .prepend {
        color: #2ca5b1;
      }

      .append {
        color: #f09630;
      }
    }
  }

  .section-content {
    font-family: Poppins, sans-serif;
    font-size: 1.25rem;
    font-style: normal;
    font-weight: 300;
    line-height: normal;
    color: #3d4753;
    text-align: center;
  }
}
</style>
