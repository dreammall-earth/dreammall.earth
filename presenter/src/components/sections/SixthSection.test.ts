import { mount } from '@vue/test-utils'
import { describe, it, expect } from 'vitest'

import SixthSection from './SixthSection.vue'

describe('renders SixthSection', () => {
  const wrapper = mount(SixthSection)

  it('renders SixthSection', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
