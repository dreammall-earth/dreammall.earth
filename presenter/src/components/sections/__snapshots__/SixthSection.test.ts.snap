// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders SixthSection > renders SixthSection 1`] = `
<div
  class="section6 v-container pa-16"
  data-v-7331c7d7=""
>
  <div
    class="v-row section-header"
    data-v-7331c7d7=""
  >
    <div
      class="v-col"
      data-v-7331c7d7=""
    >
      <h2
        class="section-headline"
        data-v-7331c7d7=""
      >
        <span
          class="prepend"
          data-v-7331c7d7=""
        >
          $t('home.section6.headline')
        </span>
        Â  
        <span
          class="append"
          data-v-7331c7d7=""
        >
          $t('home.section6.headlineAppend')
        </span>
      </h2>
    </div>
  </div>
  <div
    class="v-row section-content"
    data-v-7331c7d7=""
  >
    <div
      class="v-col"
      data-v-7331c7d7=""
    >
      <p
        data-v-7331c7d7=""
      >
        $t('home.section6.content')
      </p>
    </div>
  </div>
  <div
    class="v-row section-footer text-center"
    data-v-7331c7d7=""
  >
    <div
      class="v-col align-self-center"
      data-v-7331c7d7=""
    >
      <button
        class="v-btn v-btn--elevated v-theme--light v-btn--density-default v-btn--size-large v-btn--variant-elevated main-button main-button--primary main-button-large mt-8"
        data-v-7331c7d7=""
        data-v-9518dada=""
        style="background-color: #f09630; color: #000; caret-color: #000;"
        type="button"
      >
        
        <span
          class="v-btn__overlay"
        />
        <span
          class="v-btn__underlay"
        />
        
        <!---->
        <span
          class="v-btn__content"
          data-no-activator=""
        >
          
          
          <!--v-if-->
          <span
            class="main-button-content"
            data-v-9518dada=""
          >
            $t('home.section6.buttonText') 
            <!--v-if-->
          </span>
          
          
        </span>
        <!---->
        <!---->
      </button>
    </div>
  </div>
</div>
`;
